src/server/db/schema.ts

```diff
@@ -265,14 +290,10 @@ export const verificationTokens = createTable(
   }),
 );
 
-export const resetPasswordTokens = createTable(
-  "resetPasswordToken",
-  {
-    userId: varchar("userId", { length: 255 })
-      .references(() => users.id)
-      .primaryKey(),
-    token: varchar("token", { length: 255 }),
-    expires: timestamp("expires", { mode: "date" })
-      .notNull(),
-  }
-)
\ No newline at end of file
+export const resetPasswordTokens = createTable("resetPasswordToken", {
+  userId: varchar("userId", { length: 255 })
+    .references(() => users.id)
+    .primaryKey(),
+  token: varchar("token", { length: 255 }),
+  expires: timestamp("expires", { mode: "date" }).notNull(),
+});
```

Comment:
this was a formatting change...? I can revert this change

Reply:
this looks fine